auth_saml/views/auth_saml.xml

```diff
@@ -0,0 +1,203 @@
+<?xml version="1.0" ?>
+<odoo>
+    <!-- login form button -->
+    <template id="auth_saml.providers" name="Auth SAML Providers">
+        <t t-if="len(saml_providers) &gt; 0">
+            <em
+                t-attf-class="d-block text-center text-muted small my-#{len(saml_providers) if len(saml_providers) &lt; 3 else 3}"
+            >- or -</em>
+            <div class="o_login_saml2 list-group mt-1 mb-1 text-left">
+                <a
+                    t-foreach="saml_providers"
+                    t-as="p"
+                    class="list-group-item list-group-item-action py-2"
+                    t-att-href="p['auth_link']"
+                >
+                    <i t-att-class="p['css_class']" />
+                    <t t-esc="p['body']" />
```

Comment:
Replace all `t-esc` by `t-out`, as the former is deprecated

Reply:
Same comment as before, it is still the same as auth_oauth.

Because it is deprecated it makes sense to change it. I do not know why Odoo did not change their module.